python/lsst/ip/isr/isrMock.py

```diff
@@ -295,11 +295,109 @@ def __init__(self, **kwargs):
                                              [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
                                              [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
                                              [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0]])
-        self.bfKernel = np.array([[1., 4., 7., 4., 1.],
-                                  [4., 16., 26., 16., 4.],
-                                  [7., 26., 41., 26., 7.],
-                                  [4., 16., 26., 16., 4.],
-                                  [1., 4., 7., 4., 1.]]) / 273.0
+
```

Comment:
Please put a comment in on where these values came from.  Also, are these specific for the mock LSST camera layout?

Reply:
In fact, I think the new bfKernel should probably be put into the init for `isrMockLSST` instead.

Reply:
Ok I left the original Gaussian kernel in isrMock, and now override it with the empirical one for the simulated LSSTCam in isrMockLSST